package.json

```diff
@@ -70,7 +70,7 @@
     "circleci": "NODE_ENV=circleci npm run hint && mocha --reporter mocha-circleci-reporter --reporter-options mochaFile=$CIRCLE_TEST_REPORTS/junit/junit.xml test",
     "start": "npm run hint && node .",
     "dev": "nodemon -e js,hbs -w app",
-    "hint": "eslint '**/*.js' --quiet",
+    "hint": "eslint 'app/**/*.js' --quiet",
```

Comment:
wouldn't that break locally? We cannot make any assumption on where `app` and `api` are locally in development environment (`app` may not even be there if a developer just wants to change and test `website`)